source/keyboardHandler.py

```diff
@@ -29,6 +29,7 @@
 import core
 from contextlib import contextmanager
 import threading
+import watchdog

```

Comment:
I'm not entirely happy to introduce this dependency. Before you make any changes, I'd like to hear what @michaelDCurran has to say about the following alternative:
- Introduce a new class to the watchdog module: `watchdogObserver`
- The `watchdogObserver` can be the interface for observing the state of the watchDog. It could have a getter for the `isAttemptingRecovery` state.
- An instance of the watchdogObserver is passed into the `keyboardHandler.Initialize` function where it is stored as a protected variable on the module.